src/main/webapp/dev/js/common/datepicker.js

```diff
@@ -73,11 +75,24 @@ function prepareDatepickers() {
         showOtherMonths: true,
         gotoCurrent: true,
         defaultDate: yesterday,
-        maxDate: today,
+        maxDate: defaultStartDate,
         onSelect() {
             const newPublishDate = getMinDateForPublishDate($('#visibledate').datepicker('getDate'));
             $('#publishdate').datepicker('option', 'minDate', newPublishDate);
         },
+        onClose() {
+            const currentStartDate = $('#startdate');
+            const currentVisibleDate = $('#visibledate');
+
+            if (currentStartDate.val() === currentVisibleDate.val()) {
+                const startTime = $('#starttime');
+                const visibleTime = $('#visibletime');
```

Comment:
Those variables should be renamed to start with `$` since they refer to jQuery objects.